### PR TITLE
Idles update

### PR DESCRIPTION
**Pull request by Exr0n**
I deleted IdleVerbosely at Nikhil's suggestion because it seemed to implement completely unrelated and outdated functionality.

### CODE REVIEW

**Comment by Exr0n** [comment]
**Please do not review, as the code changes are not up to standard as defined by Daniel on slack.**

> Hey everyone, a few notes about standard updates:
> for requirements, please use the following format (its a little different than what we were doing before, but this is what Rohan and the WPILib devs do): 
```java
 @Override
  public Set<Subsystem> getRequirements() {
    return Set.of(...requirements...);
  }
```
> `...requirements...` is a Vararg, so just provide a command separated list of subsystems (or a single subsystem with no commas)
> for `initDefaultCommands`, please delete the entire method and add a javadoc to the class that the method was formerly a member of saying what command should be set as the default. Default commands are now set when the subsystem is instantiated as subsystems no longer need to have knowledge of their default commands. If you have questions about javadocs, please ask me or nikhil.
> Generally avoid wildcard imports (*) as they clutter the local namespace. Please only import what you use.
> `command.start()` is now essentially `command.schedule()`. `command.initialize()` is still `command.initialize()`.

**Comment by Exr0n** [comment]
Fixed, please check :)

File: commands/Idle.java [hunk 1]
```diff
@@ -1,46 +1,62 @@
...
+		}
+	}
 
-	@Override
-	protected void interrupted() {}
+	protected void interrupted() {
+		if (verbose) {
+			LogKitten.v("Idle " + getName() + " interrupted.");
+		}
+	}
```
Comment (NikhilSuresh24) [inline commands/Idle.java]: delete this cuz interrupted no longer exists

File: commands/Idle.java [hunk 2]
```diff
@@ -1,46 +1,62 @@
...
+	public boolean isFinished() {
+		if (verbose) {
+			LogKitten.v("Idle " + getName() + " isFinished?");
+		}
 		return false;
 	}
 
-	@Override
-	protected void end() {}
+	protected void end() {
```
Comment (NikhilSuresh24) [inline commands/Idle.java]: `end` takes in a `boolean interrupted`

Comment (NikhilSuresh24) [reply commands/Idle.java]: also call the superclass's implementation of `end`

File: commands/Idle.java [hunk 3]
```diff
@@ -1,46 +1,62 @@
...
-		setInterruptible(true);
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+
+/**
+ * Requires subsystems so no other code can interact with them.
+ */
+public class Idle extends CommandBase {
+	boolean verbose;
+	Set<Subsystem> requirements;
```
Comment (NikhilSuresh24) [inline commands/Idle.java]: delete

File: commands/Idle.java [hunk 4]
```diff
@@ -1,46 +1,62 @@
...
+
+	public Idle(String name, boolean verbose, Subsystem... subsystems) {
+		super();
+		setName(name);
+		this.verbose = verbose;
+		requirements = Set.of(subsystems);
```
Comment (NikhilSuresh24) [inline commands/Idle.java]: replace with `addRequirements(subsystems)`

File: commands/Idle.java (hunk 2 above)
Comment (bcho04) [reply commands/Idle.java]: not sure how to fix

**Pull request merged**